spec/extractors/xml/fixtures.yml

```diff
@@ -52,3 +52,6 @@
   entities.xml:
     - key: states
       string: "Alaska & Washington"
+  cdata.xml:
+    - key: foobar
+      string: "Hello <![CDATA[<b>]]>beautiful<![CDATA[</b>]]> world!"
```

Comment:
Actually would you change this string to: `"<![CDATA[Hello <b>beautiful</b> world!]]> "`
We're going to wrap the whole string in the cdata to make it more readable, and we only need the tag once as well.